src/main/java/net/minecraftforge/fmp/client/microblock/IMicroModelProvider.java

```diff
@@ -0,0 +1,21 @@
+package net.minecraftforge.fmp.client.microblock;
+
+import java.util.EnumSet;
+
+import net.minecraft.client.renderer.block.model.IBakedModel;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraftforge.fmp.microblock.IMicroMaterial;
+
+/**
+ * Interface that allows you to provide fully custom quads for a specific material.
+ */
+public interface IMicroModelProvider
+{
```

Comment:
You'd just create a new class that implements it. I guess making it an abstract class wouldn't have been an issue, but I like keeping these kinds of things as interfaces so people can mix and match them however they prefer.